google/tie
# Fix #334 - Add Modal Component

Fix to Issue #334 
* Added a styled Modal directive that should be appended to the Learner View
* Added 2 methods `showModal()` and `showExampleModal()` so that modal's contents can be changed as needed
* Added click handlers to allow user to close modal when they click `OK` button or by clicking outside of the Modal window

![screenshot 1](https://user-images.githubusercontent.com/12034267/27343713-988bed44-5598-11e7-87c2-a01cc5442025.jpeg)

# To Test
* Add the following to the bottom of the LearnerView template HTML:           
`<a href="#" ng-click="showExampleModal()">Show Example Modal</a>`
* Then click on the link and observe that a modal appears and closes as expected.

# EDIT:
I've merged in the changes to the master so that the `Privacy` button will now present a modal with the appropriate message depending on if it's browser-only or not. It should look like this:

![screenshot 2](https://user-images.githubusercontent.com/12034267/27743190-0876c712-5d70-11e7-81df-f74a6bdc3fa6.jpeg)

Behavior should otherwise be the same.
